main/keys.go

```diff
@@ -0,0 +1,87 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package main
+
+var (
```

Comment:
Nit: If these values never change they can be `const`s intead of `var`s

Reply:
Changed